packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -2096,7 +2176,56 @@ curl -v -X POST \
 }
 ```
 
-#### Assign a Custom Role to a User or Group
+#### Assign a Role to a Client Application
+
+<ApiOperation method="post" url="/oauth2/v1/clients/${clientId}/roles" />
+
+Assigns a Role to a Group
+
+##### Request parameters
+
+
+| Parameter       | Description                   | Param Type   | DataType                    | Required |
+| :-------------- | :---------------------------- | :----------- | :-------------------------- | :------- |
+| `clientId`         | ID of a Client Application | URL          | String                      | TRUE     |
+| `type`            | Type of Role to assign      | Body         | [Role Type](#role-types)    | TRUE     |
+
+##### Response parameters
+
+The assigned [Role](#role-object)
+
+##### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "type": "HELP_DESK_ADMIN"
+}' "https://${yourOktaDomain}/oauth2/v1/clients/0oa5vymVNCe2cPEeZ0g4/roles"
+```
+
+###### Response example
+
+```json
+{ 
+    "id": "JBCUYUC7IRCVGS27IFCE2SKO",
+    "label": "Help Desk Administrator",
+    "type": "HELP_DESK_ADMIN",
+    "status": "ACTIVE",
+    "created": "2023-05-01T14:24:54.000Z",
+    "lastUpdated": "2023-05-01T14:24:54.000Z",
+    "assignmentType": "CLIENT",
+    "_links": {
+        "assignee": {
+            "href": "https://{yourOktaDomain}/oauth2/v1/clients/0oa5vymVNCe2cPEeZ0g4"
+        }
+    }
+}
+```
+
+#### Assign a Custom Role to a User, Group or Client Application
```

Comment:
@vasishtajayanti-okta I can't seem to find this operation on the redocly site. Can you help?